tests/functional/test_workflow.py

```diff
@@ -448,47 +445,133 @@ def get_test_process(cls, process_id):
     @classmethod
     def retrieve_process_info(cls, process_id):
         # type: (WorkflowProcesses) -> ProcessInfo
-        base = os.getenv("TEST_GITHUB_SOURCE_URL",
-                         "https://raw.githubusercontent.com/crim-ca/testbed14/master/application-packages")
+        """
+        Retrieves the deployment, execution and package contents for a process referenced by ID.
+
+        The lookup procedure attempts multiple formats for historical reasons:
+
+            1. Look in the local ``tests.functional`` directory.
+            2. Look in remote repository:
```

Comment:
Why would we want to depend on these old repositories for testing purposes? Is it because you don't want to maintain them here and there?

Reply:
Not critical. I left them there because they were already present.
I could see new implementations though to be added for future projects, which shouldn't be included directly in Weaver.
This provides some sample repos on the expected structures if some are needed in the future.